ui-tests/src/test/java/org/openmrs/reference/DeleteRequestAppointmentTest.java

```diff
@@ -41,8 +41,8 @@ public void deleteRequestAppointmentTest() {
         ClinicianFacingPatientDashboardPage patientDashboardPage = activeVisitsPage.goToPatientDashboardOfLastActiveVisit();
         RequestAppointmentPage requestAppointmentPage = patientDashboardPage.clickOnRequest();
         requestAppointmentPage.enterAppointmentType(SERVICE_NAME);
-        requestAppointmentPage.enterValue("0");
-        requestAppointmentPage.selectUnits("Day(s)");
+        requestAppointmentPage.enterMinimumValue("0");
+        requestAppointmentPage.selectMinimumUnits("Day(s)");
```

Comment:
Refer to the comment [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/502/commits/9c172b398f03b79b816894c176df11643432a6c2#r668817026)

Reply:
any reasons for renaming this?

Reply:
> any reasons for renaming this?

The reason for renaming is [here](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/502#discussion_r668817026)